UnityProject/Assets/Scripts/Player/IBrain.cs

```diff
@@ -0,0 +1,19 @@
+using System.Collections;
+using System.Collections.Generic;
+using HealthV2;
+using UnityEngine;
+
+public interface IBrain
+{
+
+	LivingHealthMasterBase LHMB { get; set; }
+
+	virtual LivingHealthMasterBase LivingHealthMasterBase()
```

Comment:
I don't get the use of this method. You can get it from the property and implementors of the interface can modify the property as they please as well

Reply:
was just something to put into it is all wip
, and chunking the PR into smaller things